### DISCUSSION TITLE
can I have autocompletion for javascript inside html

### DISCUSSION THREAD

**Discussion by laoshaw** [neoclide/coc.nvim#3476] [Q&A] [answered]
I can use tsserver to do autocomplete for js files, sometimes I  write js inside html, can I get autocomplete for js in html too?

**Comment by oblitum** [comment, marked as answer]
[coc-html](https://github.com/neoclide/coc-html) supports this to some extent.

![gif-2021-11-20-173241](https://user-images.githubusercontent.com/1269815/142740220-1b56d80d-de9d-4d1d-b1e4-b5f2d5ab1554.gif)

**Reply by laoshaw** [reply]
you're absolutely correct. Thanks!
out of curiosity what tool do you use to capture this nice gif screenshot? is it OBS?

**Reply by oblitum** [reply]
[Custom fork](https://codeberg.org/oblitum/dotfiles/src/branch/archlinux/.local/bin/slip#L3-L5) of [slip](https://github.com/Toqozz/slip).

**Reply by laoshaw** [reply]
Thanks, just tried terminalizer and the render failed, slip is so much better!
